src/svd/field.rs

```diff
@@ -193,49 +200,27 @@ mod tests {
             </field>
             ",
             ),
-            // almost the same test but description info is missing
             (
                 Field {
                     name: String::from("MODE"),
+                    derived_from: Some(String::from("other field")),
                     description: None,
                     bit_range: BitRange {
                         offset: 24,
                         width: 2,
                         range_type: BitRangeType::OffsetWidth,
                     },
-                    access: Some(Access::ReadWrite),
-                    enumerated_values: vec![EnumeratedValues {
-                        name: None,
-                        usage: None,
-                        derived_from: None,
-                        values: vec![EnumeratedValue {
-                            name: String::from("WS0"),
-                            description: Some(String::from(
-                                "Zero wait-states inserted in fetch or read transfers",
-                            )),
-                            value: Some(0),
-                            is_default: None,
-                            _extensible: (),
-                        }],
-                        _extensible: (),
-                    }],
+                    access: None,
+                    enumerated_values: vec![],
                     write_constraint: None,
                     modified_write_values: None,
                     _extensible: (),
                 },
                 "
-            <field>
+            <field derivedFrom=\"other field\">
               <name>MODE</name>
               <bitOffset>24</bitOffset>
               <bitWidth>2</bitWidth>
-              <access>read-write</access>
```

Comment:
Why did you remove this part of the test? Also I'd rather add non-empty `derivedFrom` to the previous test: it contains more features.

Reply:
This test repeat previous almost completely. And we usually don't have other fields when have `derivedFrom`.

Reply:
Makes sense!